targets/UnrealMarketplacePlugin/examplesource/ExampleProject/BuildIOS.sh.ejs

```diff
@@ -1,27 +1,32 @@
 #!/bin/bash
-echo ========== BUILDING MAC ==========
+echo ========== BUILDING iOS ==========
 
+Usage="./BuildAndRunTestsIOS.sh <target ipa directory path> <path to project>"
+
+archivePath=$1
+projectPath=$2
+
+if [ "$#" -ne 2 ]; then
+    echo $usage
+    exit 1
+fi
+ 
 # Unreal Path, to change depending on your workspace setup.
 uePath="/Users/Shared/Epic Games"
 
 # Unreal Version, to change on every unreal version
 ueVersion="<%- ueTargetVersion %>"
 
 # Full Path. In theory it should not be edited (the engine path/version are the only ones that should be edited).
-uatPath="$uePath"/UE_"$ueVersion"/Engine/Build/BatchFiles/
+uatPath="$uePath/UE_${ueVersion}/Engine/Build/BatchFiles/"
 
 # Target Platform
 targetPlatform=iOS
 
-currentdir=`pwd`
+uprojdir="$projectPath/ExampleProject.uproject"
 
-uprojdir="$currentdir"/ExampleProject.uproject
+cd "$uatPath"
 
-currentdir=`pwd`
-
-# Destination Path, where the build will end up (relative to active directory).
-archivePath=Build/"$targetPlatform"
-pushd "$uatPath"
 # The actual build command. To adapt depending on the needs.
-. RunUAT.sh BuildCookRun -clean -rocket -installed -nop4 -nocompile -project="$uprojdir" -cook -stage -archive -archivedirectory="$archivePath" -package -clientconfig=Development -Platform="$targetPlatform" -compressed -CookAll -pak -prereqs -build -utf8output -editorrecompile
-popd
+./RunUAT.sh -ScriptsForProject="$uprojdir" BuildCookRun -nocompile -editorrecompile -installed -nop4 -project="$uprojdir" -cook -stage -archive -archivedirectory="$archivePath" -package -clientconfig=Development -ue4exe=UE4Editor -pak -prereqs -nodebuginfo -targetplatform=IOS -build -utf8output
```

Comment:
prefix should be . ./RunUAT.sh
(for now)

Reply:
What was the reason for this again?  I forget...